test/jdk/java/awt/Robot/RobotWheelTest/RobotWheelTest.java

```diff
@@ -40,22 +42,46 @@
 public class RobotWheelTest {
 
     private static final int NUMTESTS = 20;
-    private static volatile int wheelRotation;
+
+    private static AtomicInteger wheelRotation = new AtomicInteger();
+    private static int wheelSign = Platform.isOSX() ? -1 : 1;
```

Comment:
Is this the scroll rotation direction?
If it is, then as you can see in  JDK-8282716 review comments, this direction can be configured so we cannot rely on this it will always be -1 for macos...
Maybe you can do the same as it is done there to check at runtime what is the scroll direction configured?

Reply:
It is the scroll rotation direction.

Just checked, enabling/disabling `Scroll direction: Natural` for mouse or trackpad does not affect robot scrolling, so the test passes regardless of these settings.

Besides that verification of `getWheelRotation()` method itself is significant part of the test, so I doubt we can use it to detect scroll direction.

Reply:
Ok. Thanks for the clarification.